package.json

```diff
@@ -23,7 +23,7 @@
     "*.js"
   ],
   "scripts": {
-    "postinstall": "rm -f hardhat.config.ts; rm -rf airnode; git clone https://github.com/api3dao/airnode.git airnode; cd airnode && git checkout 994d895f37ca871f57224ac1603ea58e79ef071e && yarn run bootstrap && yarn build && cd ..; cp -a airnode/node_modules/@api3/airnode-node/. node_modules/@api3/airnode-node; cp -a test/hardhat.config.ts ./; rm -rf airnode",
+    "postinstall": "rm -f hardhat.config.ts; cp -a test/hardhat.config.ts ./;",
```

Comment:
I think that we could get rid of this hardhat `postinstall` script as well since there should be no more conflict with the hardhat configs. So having the hardhat config in the root should be fine (just like Airseeker) :thinking: